src/main/kotlin/no/nav/familie/ef/sak/behandling/migrering/InfotrygdPeriodeValideringService.kt

```diff
@@ -3,18 +3,36 @@ package no.nav.familie.ef.sak.behandling.migrering
 import no.nav.familie.ef.sak.infotrygd.InfotrygdService
 import no.nav.familie.ef.sak.infotrygd.InfotrygdStønadPerioderDto
 import no.nav.familie.ef.sak.infotrygd.SummertInfotrygdPeriodeDto
+import no.nav.familie.ef.sak.infrastruktur.exception.ApiFeil
 import no.nav.familie.kontrakter.ef.felles.StønadType
 import no.nav.familie.kontrakter.ef.infotrygd.InfotrygdSak
 import no.nav.familie.kontrakter.ef.infotrygd.InfotrygdSakResultat
+import org.springframework.http.HttpStatus
 import org.springframework.stereotype.Service
 import java.time.LocalDate
 import java.time.YearMonth
 
 @Service
 class InfotrygdPeriodeValideringService(
-        private val infotrygdService: InfotrygdService
+        private val infotrygdService: InfotrygdService,
 ) {
 
+    fun validerKanJournalføres(personIdent: String, stønadType: StønadType) {
+        if (infotrygdService.eksisterer(personIdent, setOf(StønadType.OVERGANGSSTØNAD))) {
+            try {
+                hentPeriodeForMigrering(personIdent)
+                // hvis hentPeriodeForMigrering returnerer noe, så betyder det att den kan migreres
+                throw ApiFeil("Denne må migreres", HttpStatus.BAD_REQUEST)
```

Comment:
Kan vi komme opp med en mer åpenbar feilmelding? 
"Det eksisterer perioder i infotrygd for denne personen. Vennligst søk opp personen og migrer før du journalfører denne journalposten" ... ? 

Ønsker vi at de skal gjøre noe slikt? ☝️

Reply:
God poeng 👍 
Ja, det er ønskelig :)